live-ingester/src/main/resources/application.yml

```diff
@@ -1,15 +1,19 @@
-application:
-  source-id: alfresco-dummy-source-id-0a63de491876
-  version: "@project.version@"
+server:
+  port: 8080
```

Comment:
I'd be surprised if this doesn't result in ports clash when running docker compose from command line.
Can we use a different port by default (or add port mapping to DC files)?

Reply:
Live ingester server port is not exposed outside container, so there shouldn't be any issues with port clashes. We expose only debug port